test/test_data/handeye/eth/eth_transform.csv

```diff
@@ -0,0 +1,4 @@
+-1.0554598e-01,  9.0038770e-01, -4.2209247e-01,  1.0752845e+03

```

Comment:
Should .csv's also be part of git LFS? Sometimes people place a lot of data in these kinds of files

Reply:
I don't think we'll ever have massive CSV files in this repo, and it's nice to get diffs if the numbers change I think.

Reply:
Ok, just leave it be